packages/web/components/Seeds/modals.tsx

```diff
@@ -131,228 +145,194 @@ export const GetRanked = () => (
-        fontWeight="bold"
-        my={4}
-      >
-        <details>
-          <summary>💸 Spending Seeds</summary>
-        </details>
-      </Box>
-      {isOpenSpending && (
-        <Flex direction="column">
-          <Text fontSize={16} fontWeight="bold" my={2}>
-            Things you can spend on now:
-          </Text>
-          <Box p={2} bgColor="whiteAlpha.300" mb={2}>
-            💡 Note: You need to be a member to access any of these
-          </Box>
-          <UnorderedList>
-            <ListItem>💸 to get a shout-out in the podcast.</ListItem>
-            <ListItem>💸 to have MetaGame tweet for you.</ListItem>
-            <ListItem>💸 to get a shout-out in the newsletter.</ListItem>
-            <ListItem>💸 to buy some limited edition merchandise.</ListItem>
-            <ListItem>
-              💸 to get products &amp; services from other players &amp; guilds.
-            </ListItem>
-            <ListItem>
-              💸 to get access to the
-              <Link
-                ml={1}
-                href="https://tokenengineeringcommunity.github.io/website/"
-                isExternal
-                color="gray.500"
-                textDecoration="underline"
-                mr={1}
-              >
-                Token Engineering
-              </Link>
-              course.
-            </ListItem>
-            <ListItem>💸 to get some user testing sessions.</ListItem>
-            <ListItem>
-              💸 to
-              <Link
-                ml={1}
-                isExternal
-                color="gray.500"
-                textDecoration="underline"
-                mr={1}
-                href="https://www.effectivealtruism.org/"
-              >
-                give away to good causes
-              </Link>
-              . ☺️
-            </ListItem>
-          </UnorderedList>
-          <Box p={2} bgColor="whiteAlpha.300" my={2}>
-            💡 Note: If you're interested in spending your Seeds on any of this,
-            ask about it in our discord channel #ask about it; #
+export const UsefulnessOfSeeds = () => (
+  <Flex direction="column">
+    <Text>
+      So far, the main thing SEEDs are used for is rewarding contributors as a
+      means of bootstrapping. Not a very promising use case in terms of token
+      value, we know!
+    </Text>
+    <Text>
+      But, we're building an entire gamified socioeconomic system, so finding
+      use cases for money shouldn't be much trouble. 😁
+    </Text>
+    <Text>
+      So, let's look at some of the current &amp; potential ways of using SEEDs!
+    </Text>
+    <CollapsableText title="💸 Spending SEEDs">
+      <Flex direction="column">
+        <Text fontSize={16} fontWeight="bold" my={2}>
+          Things you can spend on now:
+        </Text>
+        <Box p={2} bgColor="whiteAlpha.300" mb={2}>
+          💡 Note: You need to be a member to access any of these
+        </Box>
+        <UnorderedList listStyleType="💸 ">
```

Comment:
It turns out `listStyleType` is like `content` and the value has to be quoted.

```suggestion
        <UnorderedList listStyleType="'💸 '">
```